### DISCUSSION TITLE
Module not found: Can't resolve 'next/dist/client/components/request-async-storage' - in the latest 13.5.1 and 13.5.2 versions

### DISCUSSION THREAD

**Discussion by niksimFS24** [vercel/next.js#55688] [Help]
### Summary

Hello,

Does anyone knows the reason this error occurs for me when I try to upgrade to `13.5.1` or `13.5.2`:

`Module not found: Can't resolve 'next/dist/client/components/request-async-storage'` - reported in root `layout.tsx`? Even though I remove everything from `layout.tsx` and leave only `children` as returned element.

Any help? Thanks! :)

**Comment by whyosborn** [comment]
I had a similar error, but I think it was more like 'next/dist\client\components\request-async-storage'. This would happen when I built the project on a windows machine and tried to run it within a Linux docker container; however, if I built it on the Linux docker container itself, there was no issue. I don't know if it's the same issue as you had two months ago, but maybe it will help someone who had my issue.
